Index: customerservice/src/main/java/com/etiyacrm/customerservice/entities/Address.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.etiyacrm.customerservice.entities;\r\n\r\n\r\nimport com.etiyacrm.customerservice.core.entities.BaseEntity;\r\nimport jakarta.persistence.*;\r\nimport lombok.AllArgsConstructor;\r\nimport lombok.Data;\r\nimport lombok.NoArgsConstructor;\r\n\r\n@Entity\r\n@Table(name = \"addresses\")\r\n@Data\r\n@AllArgsConstructor\r\n@NoArgsConstructor\r\npublic class Address extends BaseEntity {\r\n    @Id\r\n    @Column(name=\"id\")\r\n    @GeneratedValue(strategy = GenerationType.IDENTITY)\r\n    private long id;\r\n    @Column(name=\"description\")\r\n    private String description;\r\n\r\n    @ManyToOne()\r\n    @JoinColumn(name = \"city_id\")\r\n    private City city;\r\n\r\n    @ManyToOne()\r\n    @JoinColumn(name = \"customer_id\")\r\n    private Customer customer;\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/customerservice/src/main/java/com/etiyacrm/customerservice/entities/Address.java b/customerservice/src/main/java/com/etiyacrm/customerservice/entities/Address.java
--- a/customerservice/src/main/java/com/etiyacrm/customerservice/entities/Address.java	(revision a745578cf8dbd054334debe0991689e8a639df2b)
+++ b/customerservice/src/main/java/com/etiyacrm/customerservice/entities/Address.java	(date 1715071371498)
@@ -17,6 +17,7 @@
     @Column(name="id")
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
+
     @Column(name="description")
     private String description;
 
